flutter/lib/src/sentry_flutter.dart

```diff
@@ -141,6 +142,10 @@ mixin SentryFlutter {
       integrations.add(LoadImageListIntegration(channel));
     }
 
+    if (platform.isAndroid || platform.isIOS || platform.isMacOS) {
+      integrations.add(ScreenshotIntegration());
+    }
```

Comment:
Since the screenshot is taken using the Flutter framework itself, this should work on Linux, Windows and Fuchsia as well, maybe not on Web only?

Reply:
It works on web but only when rendered via canvaskit. The html web remderer doesn't support screenshots.

Reply:
So we can use the `getRenderer` function and install it except if its `unknown` or `html`

Reply:
Yes, but I believe the renderer can be switched at runtime, so it has to be checked before doing a screenshot. Not 100% sure though.

Reply:
Should we test this manually? I'd have to setup VMs

Reply:
Added the checks for both installation and screenshot generation.